src/isnode.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+const isBrowser = ((typeof process === "undefined") || process.browser);
+const isNode = !isBrowser;
+
+module.exports.isBrowser = isBrowser;
```

Comment:
IIRC don't use camelCase anywhere in fengari; please use snake_case for consistency.